include/grpcpp/security/tls_credentials_options.h

```diff
@@ -56,7 +56,7 @@ class TlsKeyMaterialsConfig {
   int version() const { return version_; }
 
   /** Setter for key materials that will be called by the user. The setter
-   * transfers ownership of the arguments to the config. **/
+   *  takes ownership of the arguments to the config. **/
```

Comment:
By looking at the implementation, it seems the statement does not hold for `set_pem_root_certs()`.

Reply:
Do you mean it doesn't hold for `add_pem_key_cert_pair`? If so, I agree: I forgot to add `std::move` there, and have done so in the next commit.

Reply:
Sorry, Matt. I think I confused `add_pem_key_cert_pair` declared in both `TlsKeyMaterialsConfig` and `TlsCredentialReloadArg`, and mistakenly considered your comment applies to the `add_pem_key_cert_pair` defined in `TlsCredentialReloadArg`.